Cargo.toml

```diff
@@ -110,7 +110,8 @@ sha2-asm = "0.5.3"
 
 [workspace]
 members = [
-    ".", 
+    ".",
+    "libvm",
```

Comment:
is anything being removed from this file or from anywhere? How do we know it's not the old code paths that are producing the behavior.

Reply:
Moreover.. what is the observed behavior? Is the idea that you ran all the old tests and they pass? Can you add information about what was tested to the commit message for posterity?

Reply:
@lgalabru I think those are potentially confusing since we named the `stacks` specific vm module `clarity_vm`, and the more general vm module remained `vm`. Perhaps we should rename as follows:

`clarity_vm` -> `stacks_vm`
`libvm` ->  `clarity-vm-lib`

thoughts?

Reply:
Sounds good to me!

Reply:
why is it "-lib" though? most crates are "libs" but they don't have "-lib" at the end